app/src/main/java/space/taran/arkmemo/data/VersionMeta.kt

```diff
@@ -0,0 +1,8 @@
+package space.taran.arkmemo.data
+
+import space.taran.arkmemo.data.repositories.ResourceId
+
+
+data class VersionMeta(
```

Comment:
So just `ResourceId` could be used, but that will be trivial refactoring and can be done later.